devcycle.rst

```diff
@@ -292,6 +292,10 @@ Current Owners
 | Łukasz Langa         | CPython Developer in Residence | ambv            |
 +----------------------+--------------------------------+-----------------+
 
+``@python/organization-owners`` can be mentioned across the GitHub organization
+to request that one of the Organization Owners address an issue that requires
+the access levels granted soley to `permissions level <https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#permissions-for-organization-roles>`_.
```

Comment:
I don't understand this sentence. It says "Do this to request Person X address an issue that requires the access levels granted soley (sic) to permissions level."

Maybe we can split this sentence into two for clarity?